File: 백준/Bronze/1977. 완전제곱수/완전제곱수.py
import sys,math
input=lambda:sys.stdin.readline().rstrip()
m=int(input())
n=int(input())
s=[]
for i in range(m,n+1):
    t=math.sqrt(i)
    if t==int(t):
        s+=[i]
if len(s)>0:
    print(sum(s),min(s),sep="\n")
else:
    print(-1)

File: 백준/Bronze/1977. 완전제곱수/완전제곱수.c
#include <math.h>
#include <stdio.h>
int main(void) {
    int m, n, s = 0, minn;
    scanf("%d", &m);
    scanf("%d", &n);
    minn = n;
    for (int i = m; i < n + 1; i++) {
        if (sqrt(i) == (int)sqrt(i)) {
            if (minn > i) minn = i;
            s += i;
        }
    }
    if (s > 0) {
        printf("%d\n%d", s, minn);
    } else {
        printf("%d", -1);
    }
    return 0;
}